dj/api/nodes.py

```diff
@@ -567,6 +590,51 @@ def create_a_source(
     return node  # type: ignore
 
 
+@router.post("/nodes/namespace/{namespace}/", status_code=201)
```

Comment:
Do we want this under `/nodes` or can it instead be under its own endpoint `/namespaces/`?

Reply:
You're right I like that much better, changed it to `/namespaces/{namespace}/` 🚀